web-app/js/portal/search/FreeTextSearchPanel.js

```diff
@@ -116,6 +114,14 @@ Portal.search.FreeTextSearchPanel = Ext.extend(Ext.Panel, {
         }
     },
 
+    onSearchSubmit: function() {
```

Comment:
This will catch both a hit on 'enter' and a hit on the magnifier yeah? It'd be better to keep them separate for better comparison with the current analytics.